.github/workflows/ci.yml

```diff
@@ -26,8 +26,6 @@ jobs:
   tag_release:
     needs: quality_checks
     runs-on: ubuntu-latest
-    outputs:
```

Comment:
Does this want output_version_tag like in release.yml?

Reply:
added